### PR TITLE
Make it clearer that extensions can only be installed during initial setup

### LINKED ISSUE #82: Rewording extensions
### Description: 

In the docs under extensions we need to put the point forward that extensions can only be during starting. They won't be able to add it later. So instead of just extensions maybe we could call them "starter extensions" or something

### PR DESCRIPTION
**Pull request by Pabl0cks**
I've only changed the wording of the main section titles to "Starter extensions", because changing the childs titles may feel too repetitive. Also:

- Tried to enforce the idea of installation - initial project in the descriptions and subtitle.
- Added a visual hint in the main section and the `Available Extensions` section.

Closes #82

### CODE REVIEW

File: docs/extensions/extensions.md [hunk 1]
```diff
@@ -2,23 +2,29 @@
...
-This guide introduces extensions for Scaffold-ETH 2 and explains how to use and create extensions.
+This guide introduces starter extensions for Scaffold-ETH 2 and explains how to install and create them.
 
-## What are Extensions?
+## What are Starter Extensions?
 
-Extensions are modular add-ons for Scaffold-ETH 2 that provide additional functionality or serve as examples for specific features. They offer several benefits:
+Starter Extensions are modular add-ons for Scaffold-ETH 2 that provide additional functionality or serve as examples for specific features.
+
+:::info Hint
```
Comment (technophile-04) [inline docs/extensions/extensions.md:13]: Same here as https://github.com/scaffold-eth/se-2-docs/pull/83/files#r1757072217 ?

**Review by technophile-04 — COMMENTED** [review]
Tysm @Pabl0cks!! Umm not sure if "Starter Extension" seems a perfect word to me now I mean it makes sense just as word but for me it seems a bit weird while reading it in a sentence. 

Really love the idea of adding Info I think it it nicely puts out the point. 

I was thinking maybe for now just call them extension(as previously) and mention the Info? 

Maybe we can update the title/ name when we actually change the flag from `extension` to something else?  What do you think?

File: docs/extensions/availableExtensions.md [hunk 2]
```diff
@@ -4,6 +4,10 @@ sidebar_position: 10
 
 # Available Extensions
 
+:::info Hint
```
[the remark this answers is not in the document]
Comment (Pabl0cks) [reply docs/extensions/availableExtensions.md]: My initial thought was "Info" but then somehow I changed my mind and set "Hint" :)

Updated 🙏

File: docs/extensions/extensions.md (hunk 1 above)
Comment (Pabl0cks) [reply docs/extensions/extensions.md]: Updated! 🙌

**Comment by Pabl0cks** [comment]
> Tysm @Pabl0cks!! Umm not sure if "Starter Extension" seems a perfect word to me now I mean it makes sense just as word but for me it seems a bit weird while reading it in a sentence.

Agree with that! It felt a bit strange when reading it again. Changed back to Extensions and updated Hint => Info 🙌

TYSM for the feeback ♥

**Comment by technophile-04** [comment]
Tysm @Pabl0cks!!

**Pull request merged**